java/src/jmri/jmrit/logix/configurexml/WarrantManagerXml.java

```diff
@@ -163,8 +163,11 @@ static Element storeCommand(ThrottleSetting command, String type) {
         if (str==null) str = "";
         elem.setAttribute("value", str);
 
-        str = command.getBlockName();
-        if (str==null) str = "";
+        str = command.getBeanSystemName();
+        if (str==null) {
+            str = "";
+            log.error("ThrottleSetting command has no bean name! "+ command.toString());
```

Comment:
When logging, you want to avoid String concatenation in the logging statements for reasons of runtime efficiency. Our logging framework has placeholders for logging statements (and automatically calls toString() on objects), so this can be:
```
log.error("ThrottleSetting command has no bean name! {}", command);
```